Azure.ResourceManager.Core/AzureResourceManagerClient.cs

```diff
@@ -71,17 +71,17 @@ public AzureResourceManagerClient(Uri baseUri, TokenCredential credential, strin
 
         internal virtual AzureResourceManagerClientOptions ClientOptions { get; }
 
-        public SubscriptionOperations Subscription(SubscriptionData subscription) => new SubscriptionOperations(ClientOptions, subscription);
+        public SubscriptionOperations GetSubscriptionOperations(SubscriptionData subscription) => new SubscriptionOperations(ClientOptions, subscription);
```

Comment:
should return Subscription instead of SubscriptionOperations
```suggestion
        public Subscription GetSubscription(SubscriptionData subscription) => new Subscription(ClientOptions, subscription);
```